### PR TITLE
Blog post for terraform provider adoption

### PR DESCRIPTION
**Pull request by thomasdarimont**

### CODE REVIEW

File: blog/2024/terraform-provider-adoption.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+:title: Keycloak Adopts Terraform Provider
+:date: 2024-12-02
+:publish: false
+:author: Thomas Darimont
+
+# New Repository
+
+We're excited to announce that the Keycloak Terraform Provider has officially moved under the Keycloak organization! 
+You can find the new repository https://github.com/keycloak/terraform-provider-keycloak[here].
```
Comment (sschu) [inline blog/2024/terraform-provider-adoption.adoc:9]: Strictly speaking, its not a new repository. Maybe "the new repository location"?

Comment (thomasdarimont) [reply blog/2024/terraform-provider-adoption.adoc]: Fixed

File: blog/2024/terraform-provider-adoption.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+:author: Thomas Darimont
+
+# New Repository
+
+We're excited to announce that the Keycloak Terraform Provider has officially moved under the Keycloak organization! 
+You can find the new repository location https://github.com/keycloak/terraform-provider-keycloak[here].
+
+# The Journey So Far
+
+Thanks to https://www.keycloak.org/2024/09/realm-config-management-tools-survey-results.html[our community survey], we confirmed that the Keycloak Terraform Provider by mrparkers is the most widely used tool
```
Comment (vmuzikar) [inline blog/2024/terraform-provider-adoption.adoc:13]:
```suggestion
Thanks to https://www.keycloak.org/2024/09/realm-config-management-tools-survey-results.html[our community survey], we confirmed that the Keycloak Terraform Provider by [mrparkers](https://github.com/mrparkers) is the most widely used tool 
```

Comment (ahus1) [reply blog/2024/terraform-provider-adoption.adoc:13]: +1 for linking the previous maintainer. Please double-check the syntax, IMHO it is using a Markdown syntax here where it should use an AsciiDoc syntax.

**Comment by thomasdarimont** [comment]
Thanks for the review! I now mention mrparkers directly.

**Comment by thomasdarimont** [comment]
Fixed the syntax mixup. I think the blog post is now ready (and can wait for the actual terraform release).

File: blog/2024/terraform-provider-adoption.adoc (hunk 2 above)
Comment (vmuzikar) [reply blog/2024/terraform-provider-adoption.adoc]: @ahus1 Thanks for pointing out the syntax! Yeah, I believe I used an incorrect one (Markdown).

@thomasdarimont Can you please if my mistake? :)

Comment (thomasdarimont) [reply blog/2024/terraform-provider-adoption.adoc]: I already fixed the links / syntax :)

**Comment by thomasdarimont** [comment]
@vmuzikar should be good now :)

**Comment by sschu** [comment]
LGTM! @stianst Can you merge?

**Comment by ahus1** [comment]
I pushed on more change to `main` to set the flag to published, see d55efe2e2e5918b7a5bec33d7654ad1bc30ee235 

Otherwise it is only available for local preview.

**Pull request merged**